media/css/firefox/os/firefox-os-new.less

```diff
@@ -0,0 +1,897 @@
+@import "../../sandstone/lib.less";
+
+* {
+    box-sizing: border-box;
```

Comment:
We have a [box sizing mixin](https://github.com/mozilla/bedrock/blob/master/media/css/sandstone/lib.less#L243) you can probably use here